x-pack/legacy/plugins/infra/public/components/logging/log_minimap/log_minimap.tsx

```diff
@@ -135,8 +250,9 @@ export class LogMinimap extends React.Component<LogMinimapProps, LogMinimapState
   }
 }
 
-const MinimapBackground = euiStyled.rect`
-  fill: ${props => props.theme.eui.euiColorLightestShade};
+const DragTargetArea = euiStyled.rect`
+  fill: transparent;
+  cursor: move;
```

Comment:
You might want to look into using `grab` (`:hover`) and `grabbing` (`:active`) as your cursor styles for these.

https://www.w3.org/TR/css-ui-3/#valdef-cursor-grab